tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/Controllers/MatterController.cs

```diff
@@ -804,30 +804,32 @@ public IActionResult Create([FromBody] MatterMetdataVM matterMetdataVM)
         [SwaggerResponse(HttpStatusCode.OK)]        
         public IActionResult AssignContentType([FromBody] MatterMetadata matterMetadata)
         {
-            
-            ErrorResponse errorResponse = null;

```

Comment:
Instead of returning ErrorResponse, will be returning GenericResponse with IsError =true. Will be updating all the API's to return GenericResponse for both success and failure